lists2safebrowsing.py

```diff
@@ -598,11 +600,11 @@ def get_versioned_lists(config, chunknum, version):
 
         if section in WHITELIST_SECTIONS:
             ver = p_version.parse(version)
-            skip_sv_separation = (
-                ver.release[0] < VER_SV_SEPARATION_STARTED
+            skip_large_entity_separation = (
+                ver.release[0] < VER_LARGE_ENTITIES_SEPARATION_STARTED
                 and section in LARGE_ENTITIES_SECTIONS
```

Comment:
Does this `and` clause need to be here? Seems like we know we'll skip large entity separation no matter what?

Reply:
Yes, because we do want versioned standard entity lists if the version is lower than the current Nightly(74) but we want `skip_large_entity_separation` to return `True` and exit early from this function when it tries to version large entities like Google where separation is not needed.